src/components/geojson-map.vue

```diff
@@ -344,7 +361,10 @@ const selectCluster = (cluster) => {
     const extent = createEmpty();
     for (const feature of features)
       extend(extent, feature.getGeometry().getExtent());
-    fitView(extent, { duration });
+    // if zoomed way out, zoom in to level 16.
+    // if zoomed in more, keep zooming in one step (max is level 20)
+    const currentZoom = mapInstance.getView().getZoom();
+    fitView(extent, { duration, maxZoom: currentZoom > 15 ? currentZoom + 1 : 16 });
```

Comment:
> tweaked the clustering at high zoom levels to let you keep clicking on a cluster to zoom (I know this is likely to interact with Matt's work on seeing data for multiple overlapping points in a cluster so it doesnt have to get merged)

This is a nice idea. :+1: But like you said, it'll soon get replaced by what I'm working on for the cluster popup, so I think it probably makes sense to go ahead and revert. You can already zoom further using the zoom controls.